packages/mesos/extra/slave_logrotate_module.json

```diff
@@ -12,19 +12,19 @@
             },
             {
               "key": "max_stdout_size",
-              "value": "2MB"
+              "value": "10MB"
             },
             {
               "key": "logrotate_stdout_options",
-              "value": "rotate 9"
+              "value": "maxage 14"
```

Comment:
The other question is task logs.  For particularly busy services, 20 MB (the existing setting) is usually not enough.  At the same time, we aren't enforcing disk quota in Mesos, so we won't out-of-disk-kill tasks.

Technically, this change has potential for unbounded disk usage too.